config.txt

```diff
@@ -1,7 +1,8 @@
 # Default Nerves RPi 3 A+ config.txt
 #
 # It's possible to override this file by using a custom fwup.conf
-# configuration to pull in a replacement.
+# configuration to pull in a replacement. For experimental changes,
+# it's easiest to edit the file directly on the MicroSD card.
```

Comment:
I like how `cmdline.txt` says `directly on the MicroSD card via your development computer`. For whatever reason, that clicked for me thinking "o ya. mount and edit the file". What do you think about adding that here?

Reply:
Thanks - added that.